test/stresses.jl

```diff
@@ -21,6 +21,12 @@ include("testcases.jl")
         energies.total
     end
 
+    function recompute_energy2(lattice, symmetries)
+        basis = make_basis(lattice, symmetries)
+        scfres = self_consistent_field(basis; is_converged=DFTK.ScfConvergenceDensity(1e-13))
+        scfres.energies.total
+    end
+
```

Comment:
that's not a good name. Why not de fine the function inline in the `do` ... `end` block?

Reply:
Yes that seems better, inlined now